### ISSUE TITLE
MenuItem with FontAwesomeIconView(ICON) disappear after clicking it!

### ISSUE THREAD

**Issue by Yadarma** [edvin/tornadofx#869]
When I click an "item" the graphic FontAwesomeIconView will disappear. example 
`item("Patient", graphic = FontAwesomeIconView(FontAwesomeIcon.UNIVERSITY)) {
                    action { doSomething() }`

**Comment by edvin** [comment]
Please provide a complete code sample that exhibits this issue.

**Comment by edvin** [comment]
Yes, your `listview()` builder call actually replaced the graphic on the `item`. This is a good reminder of what the builders actually do :) Great that you figured it out!
